wpeverest/everest-forms
# Fix - Auto address field value not rendering issue in style

### All Submissions:


### Changes proposed in this Pull Request:
Previously, When  the Auto Address field changes by default to ‘Address' rather that others options.

### How to test the changes in this Pull Request:

1. Create a form.
2. Activate geolocation addon and drag and drop address field.
3. verify if the selected value of style is rendered correctly or not.

### Types of changes:




### Other information:




### Changelog entry

> Fix - Auto address field value not rendering issue in style.
